src/main/java/com/demo/jwt/JwtMybatisApplication/controller/mgcontroller/HostelController.java

```diff
@@ -18,14 +18,16 @@ public class HostelController {
     private HostelService hostelService;
 
     @PostMapping
+    @RolesAllowed("ROLE_ADMIN")
```

Comment:
Do we need ResponseEntity as the return type over here?

Reply:
Copy pasted the most updated project from Mybatis project, by that time changes were not rebased. Hence, there are changes on feature/dto